launcher/net/PasteUpload.cpp

```diff
@@ -36,74 +36,81 @@
+               "C:\\Users\\********\\"),  // windows
+    RegReplace(QRegularExpression("C:\\/Users\\/([^\\/]+)\\/", QRegularExpression::CaseInsensitiveOption),
+               "C:/Users/********/"),  // windows with forward slashes
+    RegReplace(QRegularExpression("(?<!\\\\w)\\/home\\/[^\\/]+\\/", QRegularExpression::CaseInsensitiveOption),
+               "/home/********/"),  // linux
+    RegReplace(QRegularExpression("(?<!\\\\w)\\/Users\\/[^\\/]+\\/", QRegularExpression::CaseInsensitiveOption),
+               "/Users/********/"),  // macos
+    RegReplace(QRegularExpression("eyJhbGciOiJIUzI1NiJ9\\.eyJ4dWlkIjoi[^\\.]+\\.[A-Za-z0-9]+", QRegularExpression::CaseInsensitiveOption),
+               "<SESSION_TOKEN>"),  // SESSION_TOKEN
+    RegReplace(QRegularExpression("new refresh token: \"[^\"]+\"", QRegularExpression::CaseInsensitiveOption),
+               "new refresh token: \"<TOKEN>\""),  // refresh token
+    RegReplace(QRegularExpression("\"device_code\" :  \"[^\"]+\"", QRegularExpression::CaseInsensitiveOption),
+               "\"device_code\" :  \"<DEVICE_CODE>\""),  // device code
+    RegReplace(QRegularExpression("(?!127\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3})(?!0\\.0\\.0\\.0)(?!1\\.[01]\\.[01]\\.1)(?!8\\.8\\.[84]\\."
+                                  "[84])(?<!version: )(?<!version )(?<!([0-9]|-|\\w))([0-9]{1,3}\\.){3}[0-9]{1,3}(?![0-9])",
+                                  QRegularExpression::CaseInsensitiveOption),
+               "**.**.**.**"),  // ipV4 (do not ask)
+    RegReplace(QRegularExpression(
+                   "(?![0:]+1?)(?<=^|\\W)((([0-9A-Fa-f]{1,4}:){7}([0-9A-Fa-f]{1,4}|:))|(([0-9A-Fa-f]{1,4}:){6}(:[0-9A-Fa-f]{1,4}|((25[0-5]|"
+                   "2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3})|:))|(([0-9A-Fa-f]{1,4}:){5}(((:[0-9A-Fa-f]{"
+                   "1,4}){1,2})|:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3})|:))|(([0-9A-Fa-f]{1,"
+                   "4}:){4}(((:[0-9A-Fa-f]{1,4}){1,3})|((:[0-9A-Fa-f]{1,4})?:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|"
+                   "1\\d\\d|[1-9]?\\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){3}(((:[0-9A-Fa-f]{1,4}){1,4})|((:[0-9A-Fa-f]{1,4}){0,2}:((25[0-5]|2["
+                   "0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:))|(([0-9A-Fa-f]{1,4}:){2}(((:[0-9A-Fa-f]{1,"
+                   "4}){1,5})|((:[0-9A-Fa-f]{1,4}){0,3}:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}"
+                   "))|:))|(([0-9A-Fa-f]{1,4}:){1}(((:[0-9A-Fa-f]{1,4}){1,6})|((:[0-9A-Fa-f]{1,4}){0,4}:((25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?"
+                   "\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:))|(:(((:[0-9A-Fa-f]{1,4}){1,7})|((:[0-9A-Fa-f]{1,4}){0,5}:((25[0-"
+                   "5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)(\\.(25[0-5]|2[0-4]\\d|1\\d\\d|[1-9]?\\d)){3}))|:)))(%.+)?(?=$|\\W)",
```

Comment:
This should be against the law

Reply:
o yeah this is yet the most complicated regex I ~wrote~ copied.
But never the less this should be a good IPV6 regex.